spec/classes/foreman_spec.rb

```diff
@@ -6,6 +6,7 @@
       let(:facts) { facts }
       let(:params) { {} }
       let(:apache_user) { facts[:osfamily] == 'Debian' ? 'www-data' : 'apache' }
+      let(:puma_workers) { [facts[:processors]['count'] * 1.5, ((facts[:memory]['system']['total_bytes'] * 1024 * 1024 * 1024) - 1.5)].min.floor }
```

Comment:
How about we override the processors and memory facts instead? I think if you modify line 6 to be something like this it should work (but I may have missed some braces somewhere):
```suggestion
      let(:facts) { override_facts(facts, processors: { count: 3 }, memory: { system: { total_bytes: SOMETHING }}) }
```